src/styles/main.scss

```diff
@@ -1,6 +1,4 @@
-html {
-  font-family: Roboto, sans-serif;
-}
+@import "utils/variables";
 
 body {
```

Comment:
you can add reset file

Reply:
Reset file connected through the link in HTML